aiobotocore/endpoint.py

```diff
@@ -36,9 +52,11 @@ def convert_to_response_dict(http_response, operation_model):
 
 
 class ClientResponseContentProxy:
-    """Proxy object for content stream of http response"""
+    """Proxy object for content stream of http response.  This is here in case
+    you want to pass around the "Body" of the response without closing the
+    response itself."""
 
-    def __init__(self, response):
+    def __init__(self, response: ClientResponse):
```

Comment:
is this function annotation ? ;) I do not see code that uses it or it is for linter?